### ISSUE TITLE
Add color keywords support

### ISSUE THREAD

**Issue by ai** [seaneking/postcss-hexrgba#22]
Can we add support for values like `rgba(black, 0.5)`? I think we should have keywords → hex mapping on npm.

**Comment by seaneking** [comment]
Yeah I think that’s still in line with the pseudo syntax we have here, so conceptually that feature fits.

Feel free to submit a PR and I’ll accept it 👍
